apis/types/types.go

```diff
@@ -106,6 +106,7 @@ type Config map[string]string
 type EnvMeta struct {
 	Name      string `json:"name"`
 	Namespace string `json:"namespace"`
+	Labels    string `json:"labels"`
```

Comment:
Why is Labels not Array?

Reply:
Because I want to use it directly as a variable to receive the `labels` flag